Dockerfile

```diff
@@ -2,15 +2,20 @@ FROM python:3
 
 WORKDIR /app
 
-ENV QUART_APP=app.py
-ENV QUART_ENV=development
-ENV PYTHONUNBUFFERED True
-ENV PYTHONPATH=/app
+ARG ARG_QUART_APP
+ARG ARG_QUART_ENV
+ARG ARG_PYTHONUNBUFFERED
+ARG ARG_PYTHONPATH
+
+ENV QUART_APP=${ARG_QUART_APP:-app.py}
+ENV QUART_ENV=${ARG_QUART_ENV:-development}
+ENV PYTHONUNBUFFERED=${ARG_PYTHONUNBUFFERED:-True}
+ENV PYTHONPATH=${ARG_PYTHONPATH:-/app}
 
 COPY ./requirements.txt .
 
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 COPY . ./
 
-CMD hypercorn -b :$PORT app:app
+CMD hypercorn -b :8080 app:app
```

Comment:
PORT should stay as GCR (Google Cloud Run) expects PORT to be specifiable (although it does default to 8080).

https://cloud.google.com/run/docs/container-contract

The ingress container within an instance must listen for requests on 0.0.0.0 on the port to which requests are sent. By default, requests are sent to 8080, but you can configure Cloud Run to send requests to the port of your choice. *Cloud Run injects the PORT environment variable into the ingress container.*

Reply:
Okay, but we don't use GCR ❓So we don't need to keep the $PORT

Reply:
Why would you not have this though? Seems to be a good practice to have a default value for PORT but allow it to be overriden too.